OrbitGl/EventTrack.cpp

```diff
@@ -18,7 +18,7 @@ EventTrack::EventTrack(TimeGraph* a_TimeGraph) : Track(a_TimeGraph) {
 
 std::string EventTrack::GetTooltip() const { return "Left-click and drag to select samples"; }
 
-void EventTrack::Draw(GlCanvas* canvas, PickingMode picking_mode) {
+void EventTrack::Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offset) {
```

Comment:
I'm torn, TBH. I was thinking about the same, because feeding the z_offset through all methods can be annoying. On the other hand, the member variable makes the scope of when this is set less clear, and it's easy to forget when adding new elements, so I opted for the parameter.